sources/academy/platform/get_most_of_actors/store_basics/how_store_works.md

```diff
@@ -104,7 +104,7 @@ Learn more about how to handle the [Issues tab](/academy/actor-marketing-playboo
 ## Resources
 
 - Best practices on setting up [testing for your Actor](https://docs.apify.com/platform/actors/publishing/test)
-- What are Apify-maintained and [Community-maintained Actors](https://help.apify.com/en/articles/6999799-what-are-apify-maintained-and-community-maintained-actors)? On ownership, maintenance, features, and support
+<!-- - What are Apify-maintained and [Community-maintained Actors](https://help.apify.com/en/articles/6999799-what-are-apify-maintained-and-community-maintained-actors)? On ownership, maintenance, features, and support -->
```

Comment:
Is this temporary? In such case we should probably note this down somewhere - even create an issue, so that we revert this change once they're up again.